src/ComponentPayload.php

```diff
@@ -81,6 +82,23 @@ public function textProperty(
         return $this->property(new TextPropertyType($names, $value, $disableEscaping));
     }
 
+    /**
+     * @param array|string $names
+     * @param string|null $value
+     *
+     * @return \Spatie\IcalendarGenerator\ComponentPayload
+     */
+    public function uriProperty(
+        $names,
+        ?string $value
+    ): ComponentPayload {
+        if ($value === null) {
+            return $this;
+        }
+
+        return $this->property(new UriPropertyType($names, $value)) ?: $this;
```

Comment:
I think this line can be changed to:

```php
return $this->property(new UriPropertyType($names, $value));
```

Or is that incorrect?

Reply:
Yes you're right.
Thanks for the correction.